docs/INSTALLATION.md

```diff
@@ -34,7 +34,7 @@ Follow the steps below to set up the repo for development
     git clone https://github.com/avantifellows/plio-frontend.git
     cd plio-frontend
     ```
-2. Make sure Docker is installed and docker version is running fine.
+2. Make sure Docker Desktop application is running and docker version is giving a proper output.
```

Comment:
The Docker Desktop part is only true for setting it up locally right? Would be good to make that segregation.